### ISSUE TITLE
Feature Request: Provide Swiss German Translations for CraftCMS (it’s easy)

### ISSUE THREAD

**Issue by phelex** [craftcms/cms#5957]
You’d have a perfect de-CH version running after a simple s/ß/ss — I kid you not.
Swiss High German (standard language) has «ss» where German has ß.
Also: Official quotation marks for languages in Switzerland are «Guillemets» – contrary to German and contrary to what we’re taught to write in school. ([Wikipedia](https://de.wikipedia.org/wiki/Anführungszeichen#Schweiz,_Liechtenstein,_Frankreich))

**Comment by brandonkelly** [comment]
I just duplicated our German translations as German - Switzerland on Crowdin. If you can review that and edit things as necessary, we can pull it into Craft!

https://crowdin.com/project/craft-cms/de-CH

**Comment by phelex** [comment]
Done

Changed «quotation marks»
Changed ß to ss
Changed wrong word “scheinbar” to ”anscheinend” (two instances)
Updated German (Source for my translation) to reflect change from “scheinbar” to ”anscheinend” (two instances)
Changed one instance of “ x ” to ” × ” because of my ego.

Have a beautiful day
Felix

**Comment by brandonkelly** [comment]
Thanks!! Craft 3.4.17 is out now with this new translation :)

**Comment by angrybrad** [comment]
@phelex Some new 3.5 strings up on https://crowdin.com/project/craft-cms/de-CH if you want to get them in. 😜

**Comment by phelex** [comment]
✅ 

Took some liberty in renaming Revisions «Überarbeitungen» (used to be “Revision”, so there's an “Über–” in there now!)
Also; does it make sense to translate the name of the templates/ directory? I don't think so – that’s why I changed it back to using original English term. Translate the word meaning “template”, yes, but going all literal translation on the file structure looks wrong to me. Maybe rephrase your sentence where you mention your “templates/” folder.

HTH

Felix

**Comment by angrybrad** [comment]
Thanks, @phelex!

**Comment by phelex** [comment]
Re: Translating to Swiss German. Try getting @ursbraem on board … he’s using Craft – and words, for that matter – more fluently than I am. That Gentleman studied German, as far as I remember his bio. Lovely guy from my hometown.
